qvm-tools/qubes-hcl-report

```diff
@@ -17,10 +17,11 @@
 # You should have received a copy of the GNU Lesser General Public
 # License along with this library; if not, see <https://www.gnu.org/licenses/>.
 
+set -uo pipefail
```

Comment:
Ok, now lets talk about `-u`, as the next hardening option that actually fired back:
```
$ qubes-hcl-report --yaml
/usr/bin/qubes-hcl-report: line 80: $1: unbound variable
```
I think that's the only erroneous case, but you may want to re-check all variables just in case.